### PR TITLE
Various nuke op tweaks and updates v2

### PR DESCRIPTION
**Pull request by ghost**
Edit: Many updates to this. See below.
- Added suit storage for nuke op space suit in shuttle 'lobby'
- Added dresser
- Added lipstick
- Added food and coffee machines
- Moved cigarette machines
- Removed multitools and power cells from nuke op lockers (moved them to the medbay/borg part of the shuttle)
- Replaced some floors for aesthetics
- Various minor changes to the shuttle

adada

### CODE REVIEW

**Comment by WJohn** [comment]
8 TC? No one's ever gonna buy that when there's already a perfectly good medkit on the shuttle.

**Comment by ghost** [comment]
The loadout in the medikit really speaks for itself, trust me, if it was any lower, everyone would buy it.

A combat hypo full of literally the most robust healing chems in the game is pretty much the only way syndi medics would be useful and there's still 2tc left over to donate to your team as well during most rounds.

**Comment by ghost** [comment]
Fine fine, TC cost reduced to 6 for the medikit.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -420,6 +420,14 @@ var/list/uplink_items = list()
 	cost = 8
 	gamemodes = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/device_tools/medic
+	name = "Tactical Medikit"
```
Comment (ChangelingRain) [inline code/datums/uplink_item.dm]: please look at lines 331 to 337, which contain a medikit that is exactly the same as this medikit, but 5 crystals

Comment (Atskadan) [inline code/datums/uplink_item.dm]: yeah it already exists i remember cheridan adding it in the nuke op update
nobody buys it

Comment (ghost) [inline code/datums/uplink_item.dm]: Well that goes to show how little it's purchased, I guess, I've played nuke ops hundreds of times and have never seen it; Figured it was leftover death squad code! Removed.

**Comment by ghost** [comment]
Medikit removed because cheridan already added it a while ago, but apparently nobody wants to buy literally the most robust drug mix in the game :aries: 

At least now that I'm aware of it, I'm going to buy the fucking shit out of it.

This is merge-ready and final because I can't think of anything else to add to spruce up the shuttle, suggestions below if you have any ideas, please.

**Comment by GunHog** [comment]
May you toss in a pAI in there? They are totally useless for Ops, but it would be MEGA fun for the pAI to tag along!

**Comment by ghost** [comment]
Good idea, gun hog.

**Comment by ghost** [comment]
WIP additions:
Storing nuke op hardsuits, masks and jet harnesses in suit storage units in a primary 'eva' room on the shuttle, and other gear being moved to a primary 'gear room' with the NVGs/bombs/toolboxes/etc.

**Comment by ghost** [comment]
ss 2014-05-10 at 07 35 55
- Ikarrus' suit storage updates properly integrated. NVGs now spawn in personal lockers. Personal lockers moved to extra gear room with belts and shit where the nvgs used to be.
- Jetpack harnesses/masks spawn in SSUs.
- Some stuff moved around, better tiling on the map, few fluff things in the base
- Medical/internals crate not empty anymore
- Domesticated space carp pet added to syndie motherbase (defanged, low ass damage, will only attack non-flukes)
- Removed white mouse from syndie motherbase I had out in the snow originally, carp goes nuts when he sees it

**Comment by ghost** [comment]
Any other suggestions for this before I declare it final?

**Comment by Atskadan** [comment]
the chair at the shuttle controls is backwards

**Comment by ghost** [comment]
FInal version for now.

**Pull request merged**